presto-docs/src/main/sphinx/presto_cpp/properties-session.rst

```diff
@@ -276,3 +276,32 @@ bytes / number of destinations for each destination before producing a Serialize
 
 Maximum number of partitions created by a local exchange.
 Affects concurrency for pipelines containing LocalPartitionNode.
+
+
+``native_spill_prefixsort_enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Formatting lines should be the same number of characters as the line being formatted.